src/emulator/kernel/src/thread/thread.cpp

```diff
@@ -208,6 +212,8 @@ bool run_thread(ThreadState &thread, bool callback) {
             break;
         }
     }
+
+    if (true) { }
```

Comment:
What's this for? Add a comment if it's not an accident.

Reply:
Nope definitely an accident! Was trying to give some ground for a breakpoint to stop at but I never used it.